static/app/components/replays/replayController.tsx

```diff
@@ -134,12 +147,29 @@ const ReplayControls = ({
   toggleFullscreen = () => {},
   speedOptions = [0.1, 0.25, 0.5, 1, 2, 4],
 }: Props) => {
+  const barRef = useRef<HTMLDivElement>(null);
+  const [compactLevel, setCompactLevel] = useState<number>(0);
```

Comment:
```suggestion
  const [compactLevel, setCompactLevel] = useState(0);
```

`number` is inferred from the default